docs/pages/includes/machine-id/machine-id-service.mdx

```diff
@@ -0,0 +1,6 @@
+```code
+$ sudo systemctl daemon-reload
+$ sudo systemctl enable machine-id
+$ sudo systemctl start machine-id
```

Comment:
Per the systemctl manual entry for `enable` ([here](https://www.man7.org/linux/man-pages/man1/systemctl.1.html)), it looks like `enable` does the same thing as `daemon-reload`. We could also consider using `enable --now` instead of `start`, though it's less explicit.

> 
           Enable one or more units or unit instances. This will create
           a set of symlinks, as encoded in the [Install] sections of
           the indicated unit files. After the symlinks have been
           created, the system manager configuration is reloaded (in a
           way equivalent to daemon-reload), in order to ensure the
           changes are taken into account immediately. Note that this
           does not have the effect of also starting any of the units
           being enabled. If this is desired, combine this command with
           the --now switch, or invoke start with appropriate arguments
           later.